elements/pfv6-alert/README.md

```
@@ -0,0 +1,35 @@
+<!-- React -->
+<Alert timeout={true}>...</Alert>
+<Alert timeout={5000}>...</Alert>
+
+<!-- Lit -->
+<pfv6-alert timeout="8000">...</pfv6-alert>
+<pfv6-alert timeout="5000">...</pfv6-alert>
+```
+
+### `component` prop
+
+**Not implemented**
```

markdownlint-cli2 (0.20.0)

[warning] Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)